DESCRIPTION

```diff
@@ -37,3 +37,16 @@ Suggests:
 VignetteBuilder: knitr
 Language: en-US
 Config/testthat/edition: 3
+Collate: 
```

Comment:
yeah, it's to make sure that `cache_environ` runs before most other things, so that later references to it work. It was generated automatically by using `@include cache.R` in `epidatr-package.R`. Default load order may or may not make it unnecessary